tox.ini

```diff
@@ -11,6 +11,8 @@ deps =
     nose
     unittest2
     protobuf==3.0.0-alpha-1
+setenv =
+    PYTHONPATH = {toxinidir}/_testing
```

Comment:
Well we can't put it in `setup.py` or in `deps` because if the install fails (from absence of system libs), then running the `tox` env will fail.

So how do we get it installed into the `tox` env?